vomsimporter.py

```diff
@@ -595,6 +615,11 @@ def resolve_cern_login_from_attributes(self, voms_user):
             return None
 
     def create_cern_sso_account_link(self, voms_user, iam_user, cern_login):
```

Comment:
You should remove `voms_user` from the list of parameters, since you have (rightly) remove it from the calls.